### PR TITLE
Ward level search

### PR DESCRIPTION
**Pull request by SooryaKumaranC-tw**
Asana Link: https://app.asana.com/0/1204322126657135/1206514083912838/f

### CODE REVIEW

File: src/features/CareViewPatients/components/CareViewPatients.jsx [hunk 1]
```diff
@@ -53,13 +89,54 @@ export const CareViewPatients = () => {
...
+    }
+  };
+
+  const noSearchResultsForPatientsWardMessage = (
+    <FormattedMessage
+      id={"NO_SEARCH_RESULTS_PATIENTS_WARD_MESSAGE"}
+      defaultMessage={"No search result available"}
+    />
+  );
+
```
Comment (ayush34957) [inline src/features/CareViewPatients/components/CareViewPatients.jsx:121]: Can we create a careviewPatientsHeader component and move search related thing to the that component?

File: src/features/CareViewPatients/components/CareViewPatients.jsx [hunk 2]
```diff
@@ -53,13 +89,54 @@ export const CareViewPatients = () => {
...
   return (
     <div className="care-view-patients-container">
       {isLoading ? (
         <Loading />
       ) : (
         <div className="care-view-patients">
           <div className="task-type">
+            <Search
```
Comment (ayush34957) [inline src/features/CareViewPatients/components/CareViewPatients.jsx:129]: same, can we move Search and filter in that header component?

File: src/features/CareViewPatients/components/CareViewPatients.jsx (hunk 1 above)
Comment (SooryaKumaranC-tw) [reply src/features/CareViewPatients/components/CareViewPatients.jsx:121]: Sure Ayush.

File: src/features/CareViewPatients/components/CareViewPatients.jsx [hunk 3]
```diff
@@ -36,46 +46,117 @@ export const CareViewPatients = () => {
...
   };
 
+  const handleKeyPress = (event) => {
+    if (event.key === "Enter") {
+      if (searchValue.length >= 3) {
+        if (!isSearched) {
+          setPreviousPage(currentPage);
+        }
+        setIsSearched(true);
+        var isCurrentPageOne = currentPage === 1;
```
Comment (ayush34957) [inline src/features/CareViewPatients/components/CareViewPatients.jsx:102]: use const instead of "var" and please check for other file changes as well

Comment (SooryaKumaranC-tw) [reply src/features/CareViewPatients/components/CareViewPatients.jsx:102]: Sure

**Review by ayush34957 — APPROVED** [review]

**Pull request merged**